modules/gdnative/include/gdnative/string_name.h

```diff
@@ -58,19 +58,12 @@ typedef struct {
 extern "C" {
 #endif
 
-void GDAPI godot_string_name_new(godot_string_name *r_dest, const godot_string *p_name);
-void GDAPI godot_string_name_new_data(godot_string_name *r_dest, const char *p_name);
-
-godot_string GDAPI godot_string_name_get_name(const godot_string_name *p_self);
-
-uint32_t GDAPI godot_string_name_get_hash(const godot_string_name *p_self);
-const void GDAPI *godot_string_name_get_data_unique_pointer(const godot_string_name *p_self);
-
-godot_bool GDAPI godot_string_name_operator_equal(const godot_string_name *p_self, const godot_string_name *p_other);
-godot_bool GDAPI godot_string_name_operator_less(const godot_string_name *p_self, const godot_string_name *p_other);
-
+void GDAPI godot_string_name_new(godot_string_name *r_dest);
+void GDAPI godot_string_name_new_copy(godot_string_name *r_dest, const godot_string_name *p_src);
 void GDAPI godot_string_name_destroy(godot_string_name *p_self);
 
+void GDAPI godot_string_name_new_with_latin1_chars(godot_string_name *r_dest, const char *p_contents);
```

Comment:
That seems to be a pretty specific survivor to the great culling :D

Reply:
Yeah, I think this is still useful. The only other way to make a `StringName` out of `const char *` is to make a `String` first, which seems wasteful when you want to, say, use it as a member name to get the value. I left `latin1` since that's the default constructor and usually identifiers fall into this range.